tests/product/tests/data/azure/azure_networking_test_cases.py

```diff
@@ -7,9 +7,81 @@
 from ..azure_test_case import AzureServiceCase
 from ..constants import RULE_PASS_STATUS, RULE_FAIL_STATUS
 
+CIS_6_1 = "CIS 6.1"
+CIS_6_2 = "CIS 6.2"
+CIS_6_3 = "CIS 6.3"
+CIS_6_4 = "CIS 6.4"
 CIS_6_6 = "CIS 6.6"
 CIS_6_5 = "CIS 6.5"
 
+cis_azure_6_1_pass = AzureServiceCase(
+    rule_tag=CIS_6_1,
+    case_identifier="test-vm-pass",
+    expected=RULE_PASS_STATUS,
+)
+
+cis_azure_6_1_fail = AzureServiceCase(
+    rule_tag=CIS_6_1,
+    case_identifier="test-vm-fail",
+    expected=RULE_FAIL_STATUS,
+)
+
+cis_azure_6_1 = {
+    "6.1 Ensure that RDP access from the Internet is evaluated and restricted expect: passed": cis_azure_6_1_pass,
+    "6.1 Ensure that RDP access from the Internet is evaluated and restricted expect: failed": cis_azure_6_1_fail,
+}
+
+cis_azure_6_2_pass = AzureServiceCase(
+    rule_tag=CIS_6_2,
+    case_identifier="test-vm-pass",
+    expected=RULE_PASS_STATUS,
+)
+
+cis_azure_6_2_fail = AzureServiceCase(
+    rule_tag=CIS_6_2,
+    case_identifier="test-vm-fail",
+    expected=RULE_FAIL_STATUS,
+)
+
+cis_azure_6_2 = {
+    "6.2 Ensure that SSH access from the Internet is evaluated and restricted expect: passed": cis_azure_6_1_pass,
```

Comment:
should it be `cis_azure_6_2_pass` ?

Reply:
isn't it?  which variable do you mean? the `cis_azure_6_2` holds both pass and fail

Reply:
Ah I see, fixed!